### PR TITLE
Duke final v7 but really

### PR DESCRIPTION
**Pull request by cmgosnell**
There is one substantive commit in here and several smaller updates. The main commit deals with disparate ownership levels of the FERC and depreciation data ([scale to fraction owned.](https://github.com/catalyst-cooperative/rmi-ferc1-eia/pull/186/commits/e2d122ae773966c7e8e6995632adb441a37d5608)). 

(also i did bad with my git management.... I made this branch off of duke_validate... I'm about to hop back over there to clean up that PR for merging so hopefully this one will be able to point to main soon.

### CODE REVIEW

**Review by zaneselvans — CHANGES_REQUESTED** [review]

File: src/pudl_rmi/connect_deprish_to_ferc1.py [hunk 1]
```diff
@@ -159,14 +161,139 @@
                 'total_fuel_cost'
             ],
         },
+    'opex_allowances':
```
[the remark this answers is not in the document]
Comment (cmgosnell) [reply src/pudl_rmi/connect_deprish_to_ferc1.py:164]: this is a great idea!!

File: src/pudl_rmi/connect_deprish_to_ferc1.py [hunk 2]
```diff
@@ -248,6 +382,116 @@ def execute(plant_parts_eia, deprish_eia, ferc1_to_eia):
...
+
+    Find the FERC records which have a different ownership % than its deprish
+    counterparts owership & the convert them.
+
+    Args:
+        scaled_fe
+        scaled_de
+        ppl
+    """
+    logger.info("doing the new thing")
```
[the remark this answers is not in the document]
Comment (cmgosnell) [reply src/pudl_rmi/connect_deprish_to_ferc1.py:409]: lol no i need to remove this. i was having import issues. good catch

File: src/pudl_rmi/connect_ferc1_to_eia.py [hunk 3]
```diff
@@ -1239,8 +1250,8 @@ def calc_annual_capital_additions_ferc1(steam_df, window=3):
     steam_df_w_addts = (
         pd.merge(
             steam_df,
-            addts[IDX_STEAM + ['capex_total', 'capex_annual_addition_rolling']],
-            on=IDX_STEAM + ['capex_total'],
+            addts[IDX_STEAM + ['capex_wo_retirment_total', 'capex_annual_addition_rolling']],
+            on=IDX_STEAM + ['capex_wo_retirment_total'],
             how='left',
```
[the remark this answers is not in the document]
Comment (cmgosnell) [reply src/pudl_rmi/connect_ferc1_to_eia.py:1255]: LOL ty!

File: src/pudl_rmi/connect_deprish_to_ferc1.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (cmgosnell) [reply src/pudl_rmi/connect_deprish_to_ferc1.py:164]: Imma use the first suggestion bc both of these field treatment dictionaries have a few quirks. And I expect we'll edit these in the future and this feels more flexible.

File: src/pudl_rmi/connect_deprish_to_ferc1.py [hunk 4]
```diff
@@ -225,6 +352,13 @@ def execute(plant_parts_eia, deprish_eia, ferc1_to_eia):
             ppl=plant_parts_eia)
     )
 
+    # scale the FERC-EIA records that we can't match to Deprish-EIA records due
+    # to ownership
+    scaled_fe = scale_to_fraction_owned(
+        scaled_de=scaled_de,
+        scaled_fe=scaled_fe,
```
[the remark this answers is not in the document]
Comment (cmgosnell) [reply src/pudl_rmi/connect_deprish_to_ferc1.py:359]: yeah this is fair! I think I'd like to remove the `_to_`'s. the modules have `_to`'s but they are action-oriented... so I think the tables themselves can be just `dataset1_dataset2`

File: src/pudl_rmi/connect_deprish_to_ferc1.py [hunk 5]
```diff
@@ -225,6 +352,13 @@ def execute(plant_parts_eia, deprish_eia, ferc1_to_eia):
...
+        ppl=plant_parts_eia
```
[the remark this answers is not in the document]
Comment (cmgosnell) [reply src/pudl_rmi/connect_deprish_to_ferc1.py:360]: "EIA plant-parts list"

**Review by zaneselvans — APPROVED** [review]

**Pull request merged**